docs/GettingStarted.md

```diff
@@ -441,10 +441,10 @@ To capture a trace in a file use the following commands:
 
 1. Start tracing:
    ```cmd
-   wpr.exe -start "%ProgramFiles%\ebpf-for-windows\ebpfforwindows.wprp" -filemode
+   wpr.exe -start "%ProgramFiles%\[eBPF for Windows install folder]\ebpfforwindows.wprp" -filemode
```

Comment:
eh... This _could_ become an irritant to the end user.  If unattended installs _are_ possible, how does it work? does the user have to confirm/supply the install location folder? If not, how would the user know what the install folder is?

Even in case of interactive installs, this would require folks to remember the install location when they actually _need_ to gather the traces (could be days after the install).  IMO a better option would be to pick a default location and hardcode it in our docs with an explicit note to change that name to the custom location.  This way, the command would work 'out-of-the-box' for folks who go with the defaults (as most would) and a helpful hint to unblock folks that go the custom route.

Reply:
> eh... This _could_ become an irritant to the end user. If unattended installs _are_ possible, how does it work? does the user have to confirm/supply the install location folder? If not, how would the user know what the install folder is?

Unattended installs *are* possible, and it's documented in `InstallEbpf.md`.

> 
> Even in case of interactive installs, this would require folks to remember the install location when they actually _need_ to gather the traces (could be days after the install). IMO a better option would be to pick a default location and hardcode it in our docs with an explicit note to change that name to the custom location. This way, the command would work 'out-of-the-box' for folks who go with the defaults (as most would) and a helpful hint to unblock folks that go the custom route.

I believe that users of eBPF will have to anyways create their own scripts for different purposes within their individual contexts. IMHO, having a clear statement showing that the installation folder *is* customizable, will avoid confusion/errors from just cut & pasting from the docs.
@Alan-Jowett @dthaler @dahavey for opinions.

Reply:
IMO ebpf devs (or any dev for that matter) are expected to be responsible for their _own_ scripts.  My point was about _our_ scripts and/and or suggested command lines.  If documented, IMO they _ought_ to work 'as-is' for default installs. But this is hardly an issue to hold up this PR :-)

Reply:
As @dv-msft suggested, should we resolve this and fork this into a separate doc issue?

Reply:
Ok, I renamed the default install folder back to `ebpf-for-windows` with reference in the docs, as well as the MSI, so to match any references (i.e. recent PRs).
This way we can eventually take this topic to a separate discussion, i.e. product finalization.